### ISSUE TITLE
Sample app?

### ISSUE THREAD

**Issue by rogerroelofs** [RobinBuschmann/express-soap#3]
Thanks so much for this module!  I'm having a little trouble figuring out how to integrate this into my app.  Is there a working demo somewhere that I could look at to see where I might be going wrong?  I'm guessing my problem is that I'd like to confine the soap code to it's own controller and I probably don't have things set up properly.   Playing with actual working code would help me understand how the pieces fit together.

**Comment by rrubio** [comment]
+1

**Comment by nkgobane** [comment]
Hi there. This issue has still not been resolved? I am battling to understand how to integrate this into my code. Please assist

**Comment by jonnytest1** [comment]
i dont think this package is a code generator instead it jsut parses the request and handles the response - but the services itself stil lhave to be manually added 

(i mean it could at least log out if a service/port/method is missing - but i guess thats what the log attribute is for)
